test/fixtures/fake_project/mix.exs

```diff
@@ -0,0 +1,30 @@
+defmodule FakeProject.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :fake_project,
+     version: "0.0.1",
+     elixir: "~> 1.1-dev",
```

Comment:
This fails to compile in Travis. Could you change the Elixir requirement to `~> 1.0`?